10_dynamic_regression_models.Rmd

```diff
@@ -217,8 +217,137 @@ insurance %>%
 ```
 
 
+## Forecast with different levels of TVadverts
 
+Set new data to 20 year-month period time-frame:
+```{r eval=FALSE}
+new_data(insurance, 20)
+```
+
+
+Sample `TVadverts` ranging from 7 to 10:
+```{r}
+set.seed(1234)
+sample <- sample(7:10,20,replace=T)
+```
+
+
+Create a newdata with sampled TVadverts:
+```{r}
+insurance_future2 <- new_data(insurance, 20) |>
+  mutate(TVadverts = sample)
+insurance_future2%>%head
+```
+
+Fit the model on sampled TVadverts and plot it:
+```{r}
+fit_best |>
```

Comment:
Starting here you reference `fit_best`, but it doesn't exist. Is it supposed to have been created at some point above here? The book won't build with failing code.

Reply:
updated chapter